packages/core/src/utils/fetchStoreData.ts

```diff
@@ -0,0 +1,8 @@
+import { TConfig } from '../types'
+
+export default async function fetchStoreData() {
+  const response = await fetch('/api/config')
```

Comment:
This stuff should probably continue to live within the `packages/editor` context as it's only relevant to the package, and not for `web`. Trying to reserve `packages/core` for stuff relevant to both!

Reply:
Okay understood will make the change!

Reply:
Making this change means moving the core/src/component/Layout.tsx back to the editor. This change causes the sidebar to lose its collapsibility. As there seems to be a problem with its type definition.